app/pb/threads.proto

```diff
@@ -54,6 +54,9 @@ message GetThreadRes {
   // A token that can be sent as `page_token` to retrieve the next page.
   // If this field is omitted, there are no subsequent pages.
   string next_page_token = 2;
+
+  // number of responses (comments and replies) associated with the thread
+  uint32 num_responses = 3;
```

Comment:
Please move this above `next_page_token`: for consistency, it would be best for that to be last. You can either renumber of leave this as `3`